solidity/test/DepositFundingTest.js

```diff
@@ -155,6 +160,78 @@ describe("DepositFunding", async function() {
         "Opening new deposits is currently disabled.",
       )
     })
+
+    it("respects the supply cap schedule", async () => {
+      const bn = web3.utils.toBN
```

Comment:
I'll say it bugs me just a touch that you've special-purposed a helper here, but this is such a pervasive issue that I understand why you didn't bring the hammer down :laughing:

Reply:
It's the worst thing. I can't stand `BN`